libs/feature/dataviz/src/lib/service/data.service.ts

```diff
@@ -148,6 +154,34 @@ export class DataService {
     )
   }
 
+  async getDownloadLinksFromOgcApiFeatures(
+    ogcApiLink: DatasetServiceDistribution
+  ): Promise<DatasetDistribution[]> {
+    const apiUrl =
+      ogcApiLink.url.href.slice(-1) === '?'
+        ? ogcApiLink.url.href.slice(0, -1)
+        : ogcApiLink.url.href
```

Comment:
Actually, this is the part that needs an upgrade from `ogc-client` :slightly_smiling_face: 
The lib will soon handle URLs with questions marks at their end, so this will be removed. But good suggestion though!